pages/Messages/Validation.md

```diff
@@ -0,0 +1,173 @@
+---
+name: Validation
+route: /Messages/Validation
+menu: Messages
+---
+
+# Message Validation
+
+All messages read from an archive should be validated to guarantee the message's authenticity, authorization and legality.
+Messages should be validated before being written to an archive file, however clients should not rely on this being the case and should perform their own validation on messages in the interest of protection against malicious actors.
+
```

Comment:
It just occurred to me that there might be legal liability ramifications in making statements like this and the one above.  We might want to submit this to PL's legal department for review.  I think we're going to need a ToS soon with a "no warranty" exclusion sort of thing.